text/0060-tasks-api.md

```diff
@@ -678,8 +678,8 @@ This part demonstrates keyset paging in action on `/tasks`. The items `uid` rema
 
 ###### 10.5.3. Errors
 
-- 🔴 Sending a value with a different type than `Integer` for `limit` returns a [bad_request](0061-error-format-and-definitions.md#bad_request) error.
-- 🔴 Sending a value with a different type than `Integer` for `from` returns a [bad_request](0061-error-format-and-definitions.md#bad_request) error.
+- 🔴 Sending a value with a different type than `Integer` for `limit` returns a [bad_request](0061-error-format-and-definitions.md#bad_request) error. //TODO
```

Comment:
@irevoire We have missed some `bad_request` code (mostly GET parameters for Index, Documents, Tasks and API keys pagination). Marking them with `//TODO`.

Reply:
Fixed with `invalid_task_{limit|from}`